R/check_code.R

```diff
@@ -30,16 +34,22 @@
 #' @template learnr_args
 #' @template glue_correct
 #' @template glue_incorrect
-#' @param glue_pipe A glue string that returns the final message displayed when a user uses a pipe,
+#' @param glue_pipe A glue string that returns the final message displayed when
+#'    the studentuses a pipe,
```

Comment:
missing space.

are we doing `student` or `user`?

Reply:
using student/instructor in the documentation (internal code variables may need to be renamed eventually) since there were a few times when the "user" would also refer to the instructor writing the document.

Reply:
Ok.

Still missing space in `studentuses`